src/sdk.ts

```
@@ -73,12 +96,80 @@ export class BiometrySDK {
+    const response = await this.request<{ data: { is_consent_given: boolean; user_fullname: string } }>(
+      '/api-consent/strg-consent',
+      'POST',
+      body,
+      headers
     );
 
+    return {
+      is_consent_given: response.data.is_consent_given,
+      user_fullname: response.data.user_fullname,
+    };
     return response;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)